src/pages/Login/Login.js

```diff
@@ -2,16 +2,28 @@ import React, { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
 import Modal from 'react-modal';
-
 const { Kakao } = window;
 
 function Login() {
   const [modalOpen, setModalOpen] = useState(false);
+  const [profile, setProfile] = useState();
 
+  //modal
   const modalHandler = () => {
     setModalOpen(prev => !prev);
   };
 
+  //token
+  const logOutHandler = () => {
+    setModalOpen(prev => !prev);
```

Comment:
```suggestion
    modalHandler()
```

여기서 하는 일은 모달의 상태를 변경하고, 로그아웃을 한다 인 것 같네요
그러면 setModalOpen이라는 setState 함수를 바로 호출하기보다는 이미 모달을 컨트롤하기 위해 만들어둔 modalHandler 함수를 호출하는게 좋을 것 같네요!!

이렇게 하면 추후에 modal을 handling하는 로직에 변경이 생겼을때 (단순히 이름변경부터, 여러개의 state를 수정하는식으로 변경된다던가 여러 케이스가 있겠네요) modalHandler 함수 내부의 동작만 바꿔주면 되서 좀 더 변화에 유연하게 대응할 수 있습니다

Reply:
넵 리뷰 반영 완료했습니다~~